src/lib/include/atan.h

```diff
@@ -0,0 +1,13 @@
+#ifndef atan_h
+#define atan_h
+
+#include <immintrin.h>
+
+#ifdef __AVX2__
```

Comment:
This works but we should control what SIMD level is emitted with CMake. Working on that in #22 so nothing to do here.